install.R

```diff
@@ -0,0 +1,7 @@
+install.packages("lubridate")
```

Comment:
Does it do any harm to run these commands twice? If no we could keep them in their original location as well?

Reply:
Hi Tim! Sorry, this got completely buried in my other notifications when I was travelling!  This is awesome and I'll merge it in!

Reply:
No worries :)